frontend/public/kubevirt/components/vm/menu-actions.jsx

```diff
@@ -41,16 +50,19 @@ const menuActionStart = (kind, vm) => ({
   }),
 });
 
-const menuActionRestart = (kind, vm) => ({
-  hidden: !_.get(vm, 'spec.running', false),
-  label: 'Restart Virtual Machine',
-  callback: () => restartVmModal({
-    kind,
-    resource: vm,
-  }),
-});
+const menuActionRestart = (kind, vm, actionArgs) => {
+  return {
+    hidden: !_.get(vm, 'spec.running', false) || isImporting(vm, findImporterPods(actionArgs[PodModel.kind], vm)),
```

Comment:
please check for the presence of actionArgs. We just found out about this issue (https://github.com/kubevirt/web-ui/pull/246) in overview page where it will not be working.

Reply:
Done!